src/ray/core_worker/core_worker.cc

```diff
@@ -15,7 +16,8 @@ CoreWorker::CoreWorker(
       raylet_socket_(raylet_socket),
       log_dir_(log_dir),
       worker_context_(worker_type, job_id),
-      io_work_(io_service_) {
+      io_work_(io_service_),
+      heartbeat_timer_(io_service_, boost::asio::chrono::milliseconds(100)) {
```

Comment:
let's make the "100" here (and down below) a configuration argument. I'd also consider bumping it to 500ms and the number of objects to 5000.

Reply:
Changed to take it from the config. I think we should keep it the same as the raylet heartbeat.

Reply:
We're sending at 500ms intervals now.